.github/workflows/frontend.yaml

```diff
@@ -0,0 +1,52 @@
+name: frontend
+on:
+  push:
+    branches:
+      - main
+      - r[0-9]+ # Trigger builds after a push to weekly branches
```

Comment:
I don't think that is necessary anymore and if it would be the correct path would be more like `weekly/f[0-9]+`

```suggestion
```